### PR TITLE
Balance

### PR DESCRIPTION
**Pull request by dcsdprogrammers**
Allows robot to autonomously get itself on ramp and balance on it. adjusted controls to stop numbers from being inverted for auto commands.

### CODE REVIEW

**Review by Zandar-Double07 — APPROVED** [review]
Looks Good

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -7,57 +7,72 @@
...
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.Command;
-
-import frc.robot.commands.BasicDrive.DriveBackward;
-import frc.robot.commands.BasicDrive.DriveForward;
-import frc.robot.commands.BasicDrive.TurnLeft;
-import frc.robot.commands.BasicDrive.TurnRight;
+import frc.robot.commands.BasicDrive.*;
+import frc.robot.commands.BalanceSteps.*;
+import frc.robot.commands.BalanceSteps.DriveTillDegrees;
```
Comment (Atika2022) [inline src/main/java/frc/robot/RobotContainer.java]: Delete this as it's redundant

**Review by Atika2022 — APPROVED** [review]
The changes in the code seem alright and it works!

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (Zandar-Double07) [reply src/main/java/frc/robot/RobotContainer.java]: Done

**Review by superheropug — APPROVED** [review]

**Pull request merged**